src/index.js

```diff
@@ -1 +1,57 @@
-import './style.css';
\ No newline at end of file
+import './style.css';
+import getMovie from './modules/getMovies.js';
+import display from './modules/display.js';
+import getLikes from './modules/getLikes.js';
+import like from './modules/like.js';
+
+const appId = 'WGv1nO0NTRyk5wZp7rJP';
+
+const numberOfMovies = 6;
+let index = 1;
+const movies = [];
+while (index <= numberOfMovies) {
+  const movie = await getMovie(index + 1);
+  movies.push(movie);
+  index += 1;
+}
+
+const updateLikes = async (appId) => {
+  let likes = await getLikes(appId);
+
+  if (likes.length === 0) {
+    likes = [];
+  } else {
+    likes = JSON.parse(likes);
+  }
+  return likes;
+};
+
+const likes = await updateLikes(appId);
+//  console.log(likes);
```

Comment:
it is better to remove commented codes for the best practice and for a clean code. ☺️

Reply:
Thank you @yashodhicy I will make the changes.